lib/inquisitor.ex

```diff
@@ -106,4 +113,15 @@ defmodule Inquisitor do
     |> List.last()
     |> Mix.Utils.underscore()
   end
+
+  @doc false
+  def handle_non_integer(i) do
```

Comment:
This returns 0 if the value is an integer.

For example, `conn = get build_conn, "/foo", [limit: 2]` results in a limit of 0.